app/src/main/java/com/example/soptexampleproject/presentation/home/screens/ViewPagerActivity.kt

```diff
@@ -0,0 +1,61 @@
+package com.example.soptexampleproject.presentation.home.screens
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import androidx.fragment.app.Fragment
+import androidx.viewpager2.widget.ViewPager2
+import com.example.soptexampleproject.R
+import com.example.soptexampleproject.databinding.ActivityViewPagerBinding
+import com.example.soptexampleproject.presentation.home.screens.fragment.FragmentChangeAdapter
+import com.example.soptexampleproject.week3.Fragment.PagerFragmentList
+import com.example.soptexampleproject.week3.Fragment.PagerFragmentProfile
+import com.example.soptexampleproject.week3.Fragment.PagerFragmentSetting
+
+class ViewPagerActivity : AppCompatActivity() {
+    private lateinit var adapter: FragmentChangeAdapter
+    private val myFragments =
+        listOf<Fragment>(PagerFragmentProfile(), PagerFragmentList(), PagerFragmentSetting())
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        ActivityViewPagerBinding.inflate(layoutInflater).run {
+            setContentView(root)
+            bindingView(this)
+        }
+    }
+
+    private fun bindingView(binding: ActivityViewPagerBinding) {
+        adapter = FragmentChangeAdapter(this, myFragments)
+        binding.myViewPager.adapter = adapter
+        binding.myNavi.setOnItemSelectedListener {
+            when (it.itemId) {
+                R.id.menu_android -> {
+                    binding.myViewPager.currentItem = FIRST_FRAGMENT
+                    return@setOnItemSelectedListener true
+                }
+                R.id.menu_list -> {
+                    binding.myViewPager.currentItem = SECOND_FRAGMENT
+                    return@setOnItemSelectedListener true
+                }
+                else -> {
+                    binding.myViewPager.currentItem = THIRD_FRAGMENT
+                    return@setOnItemSelectedListener true
+                }
+            }
+        }
```

Comment:
```suggestion
        binding.myNavi.setOnItemSelectedListener {
            when (it.itemId) {
                R.id.menu_android -> binding.myViewPager.currentItem = FIRST_FRAGMENT
                R.id.menu_list -> binding.myViewPager.currentItem = SECOND_FRAGMENT
                else -> binding.myViewPager.currentItem = THIRD_FRAGMENT
            }
            return@setOnItemSelectedListener true
        }
```

다음과 같이 중괄호를 생략해서 코드를 작성할 수 있습니다 선생님

Reply:
한줄에는 생략가능 감사합니닷!